rfc-drafts/stream.md

```diff
@@ -183,6 +203,21 @@ while let Some(v) = stream.next().await {
 }
 ```
 
+We could also consider adding a try_next? function, allowing
```

Comment:
I had to [look this up](https://docs.rs/futures/0.3.5/futures/stream/trait.TryStreamExt.html#method.try_next) to remember why it was useful, so it might be worth linking to the futures-rs version.

Reply:
I'd also expect to see some reasoning on why this isn't being proposed by the RFC yet. We want to stick to just the bits everyone needs for now, or something along those lines.